src/Sources/Coalescence.jl

```diff
@@ -12,131 +12,171 @@ using Cloudy.ParticleDistributions
 using Cloudy.KernelTensors
 using Cloudy.EquationTypes
 using QuadGK
+using RecursiveArrayTools
 
 
 # methods that compute source terms from microphysical parameterizations
-export get_int_coalescence
 export update_coal_ints!
 export initialize_coalescence_data
 
 """
-get_int_coalescence(::OneModeCoalStyle, mom_p::Array{Real}, par::Dict, ker::KernelTensor{Real})
+update_coal_ints!(::AnalyticalCoalStyle, kernel::KernelTensor{FT}, pdists::Array{ParticleDistribution{FT}}, thresholds::Vector{FT}, coal_data::NamedTuple)
```

Comment:
I don't quite understand what thresholds refers to here...

Reply:
thresholds are used for computing S. In my computations I assume a heaviside weight function with the given thresholds.